src/preact/utils/docInfo.js

```diff
@@ -0,0 +1,46 @@
+import {urlUtils} from './url';
+
+export const docInfo = {
+  /**
+   * @param {string} metaName
+   * @return {string|null}
+   */
+  getMetaByName(metaName) {
+    const metas = self.document.head.querySelectorAll('meta[name]');
+    return (
+      Array.from(metas.values())
```

Comment:
No need for `vaules()` here, `Array.from()` will handle a NodeList just fine.
```suggestion
      Array.from(metas)
```